ufo/models.py

```diff
@@ -5,25 +5,26 @@
 LONG_STRING_LENGTH = 1024
 
 class Model(db.Model):
-  __abstract__ = True
+  """Helpful functions for the database models
 
-  @classmethod
-  def GetAll(cls):
-    return cls.query.all()
+  Most method implementations are taken from
+  https://github.com/sloria/cookiecutter-flask"""
+  __abstract__ = True
 
-  @classmethod
-  def GetById(cls, id):
-    return cls.query.get(id)
+  def update(self, commit=True, **kwargs):
+    for attr, value in kwargs.items():
+      setattr(self, attr, value)
+    return commit and self.save() or self
 
-  def Add(self):
+  def save(self, commit=True):
     db.session.add(self)
-    db.session.commit()
-
+    if commit:
+      db.session.commit()
     return self
 
-  def Delete(self):
+  def delete(self, commit=True):
     db.session.delete(self)
-    db.session.commit()
+    return commit and db.session.commit()
```

Comment:
It's done in the more verbose way in `save` because we want to return the current object no matter what, this is just a shorter way to write it (and also what I copied from the skeleton as noted above).

I'd be fine with switching it to be more verbose if you would like to do that.